### ISSUE TITLE
Typescript compile errors hidden

### ISSUE THREAD

**Issue by michaelstone** [sth/karma-summary-reporter#11]
Hey, I'm not sure if this is effectively a duplicate of the previous discussions about printing errors, but wanted to raise it because I had a tough time tracking down the cause. 

I was using this on an angular project and if there's a typescript compilation error, the output from Karma is that it can't connect to Chrome. This makes sense in hindsight because the bundle wasn't able to be built but wasn't obvious since usually the TS errors are visible.

e.g. this is the output from a basic angular project (generated with ng new) with a small typescript error added. The second is with the summary reporter added.

![tserror](https://user-images.githubusercontent.com/2778546/74419453-d6189f00-4e41-11ea-8605-085be50e956b.jpg)

![hiddenerror](https://user-images.githubusercontent.com/2778546/74419455-d7e26280-4e41-11ea-92ad-6f0e0f424874.jpg)

**Comment by sth** [comment]
This missing error message is not how it is supposed to work. While summary-reporter itself doesn't print the messages, it shouldn't suppress messages printed by someone else. Adding a the summary reporter should only add a summary table, not remove output created by some other component.

Also there seems to be a deeper problem: In the first screenshot Chrome actually *did* connect, despite the Typescript error. Only after adding summary-reporter the browser can't be captured anymore!

This is surprising, since summary-reporter doesn't do anything directly with the browser connection. It just processes messages karma passed on to it. Something must go horribly wrong when karma tries to process the typescript error with summary-reporter. Likely some exception is thrown that causes internal problems in karma.

**Comment by sth** [comment]
So in this situation the reporter's `onRunComplete()` was called without a call to `onRunStart()` first, which was not expected and resulted in an exception.

Thanks for the clear bug report with the screen shots and instructions to reproduce the problem, that was really helpful!

I published a new version **v1.7.2** with the fix.

**Comment by michaelstone** [comment]
nw, thanks for the quick fix!
